File: Architecture.Core/Entities/ProfMovementInfo.cs
using Architecture.Core.Entities.Core;
using Architecture.Core.Entities.LU;
using System;

namespace Architecture.Core.Entities
{
    public class ProfMovementInfo : Auditable
    {
        public int MovementInfoId { get; set; }
        public int ProfileId { get; set; }
        public virtual ProfBasicInfo ProfBasicInfo { get; set; }
        public int? CountryNameId { get; set; }
        public virtual CountryName CountryName { get; set; }
        public DateTime StartDate { get; set; }
        public DateTime EndDate { get; set; }
        public string Purpose { get; set; }
        public string Status { get; set; }

    }
}

File: Architecture.Web/ClientApp/src/app/Shared/Entity/ClientProfile/profMovementInfo.ts
import { Data } from '@angular/router';
import { Auditable } from '../Common/auditable';

export class profMovementInfo extends Auditable {
    public MovementInfoId: number;
    public profileId: number;
    //public virtual ProfBasicInfo ProfBasicInfo { get; set; }

    //    public int ? CountryNameId { get; set; }
    //    public virtual CountryName CountryName { get; set; }
    public countryName: string;

    public countryNameId: number;

    public purpose: string;

    public status: string

    public startDate: Date;

    public endDate: Date;

}


